.github/workflows/dest_duckdb.yml

```diff
@@ -0,0 +1,83 @@
+name: Destination Plugin DuckDB Workflow
+
+on:
+  pull_request:
+    paths:
+      - "plugins/destination/duckdb/**"
+      - ".github/workflows/dest_duckdb.yml"
+  push:
+    branches:
+      - main
+    paths:
+      - "plugins/destination/duckdb/**"
+      - ".github/workflows/dest_duckdb.yml"
+
+jobs:
+  plugins-destination-duckdb:
```

Comment:
Do we have an image for duckdb to run against?

Reply:
Oh yeah, I haven't updated the workflow yet, thanks

Reply:
Actually, what am I saying 🤦 We don't need an image, because the tests run against an in-memory database